reflex/reflex.py

```diff
@@ -452,6 +452,191 @@ def export(
     )
 
 
+@cli.command()
+@loglevel_option
+@click.option(
+    "--backend-url",
+    type=str,
+    default="http://localhost:8000",
+    help="The URL of the backend server.",
+)
+@click.option(
+    "--frontend-url",
+    type=str,
+    default="http://localhost:3000",
+    help="The URL of the frontend server.",
+)
+@click.option(
+    "--delay",
+    type=int,
+    default=0,
+    help="The delay (in milliseconds) to wait before taking a screenshot.",
+)
+@click.option(
+    "--full-page/--no-full-page",
+    default=False,
+    is_flag=True,
+    help="Whether to capture the full page or just the viewport.",
+)
+@click.option(
+    "--width",
+    type=int,
+    default=1280,
+    help="The width of the screenshot.",
+)
+@click.option(
+    "--height",
+    type=int,
+    default=720,
+    help="The height of the screenshot.",
+)
+@click.option(
+    "--headed/--headless",
+    default=False,
+    is_flag=True,
+    help="Whether to run the browser in headed mode (visible) or headless mode (invisible).",
+)
+@click.option(
+    "--stateful/--stateless",
+    default=True,
+    is_flag=True,
+    help="Whether to run the browser in stateful mode (preserving session) or stateless mode (fresh session).",
+)
+@click.option(
+    "--output-dir",
+    type=Path,
+    default=".",
+    help="The directory to save screenshots to.",
+)
+def take_screenshots(
```

Comment:
i think this function should exist outside of the cli so it can also be called as a library